crates/runtime/src/accelerated_table.rs

```diff
@@ -434,18 +452,27 @@ impl AcceleratedTable {
                     }
                 }
                 AccelerationRefreshMode::Full(receiver) => {
+
+                    let refresh_params_lock = refresh_params.read().await;
+                    let refresh_settings = &*refresh_params_lock;
```

Comment:
Explicitly calling `.deref()` is clearer imo. You will need to add `use std::ops::Deref` to the top of the file.

Reply:
Yeah, I like `.deref` more as well, but lint does not  - https://rust-lang.github.io/rust-clippy/master/index.html#explicit_deref_methods

Should I update back to deref and disable corresponding lint warning?

Reply:
TIL - no we can keep the clippy style then.